andela/suyabay
# Regular users should be able to request for premium accounts.

#### What does this PR do?
- Create a form where regular users can request for a premium account.
#### Description of Task to be completed?
- Create a model for account upgrade
- Write method for view and add user requests for premium account
- Add account upgrade model to factory
- Create a blade file for account upgrade request and view all user requests.
- Write tests for regular users request for a premium account.
#### How should this be manually tested?
-  A regular user should see a link to request for a premium account at the right corner of the website, and when they click on it, they are directed to a form.
- The user should fill the form, and if successful, they should see a message `Your request was submitted, we will get back to you soon!`.
- On the admin part, when they are logged in, they should user requests notifications at the top nav bar of the current web page.
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#### Screenshots (if appropriate)
#### Questions:

[@unicodeveloper](https://github.com/unicodeveloper)
[@andela-oadebayo](https://github.com/andela-oadebayo)
